.github/workflows/scala3-build.yml

```diff
@@ -53,7 +53,7 @@ jobs:
 
       - name: Compile and run tests on Scala 3
         env:
-          DEVELOCITY_ACCESS_TOKEN: ${{ secrets.GE_ACCESS_TOKEN }}
+          DEVELOCITY_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
```

Comment:
The parameter variable should be key too, better to keep the same

Reply:
The `GE_ACCESS_TOKEN` secret is controlled by ASF Infra and in use by many other projects. At some point, we will migrate the name, but it would be a larger effort to do that, given other projects use the secret.

Reply:
secrets.GE_ACCESS_TOKEN is what is setup by the ASF Infra team - we have no control over this

DEVELOCITY_ACCESS_KEY is the name expected by Develocity - we have no control over this

Reply:
Thanks for the update , it's much cleaner now